SwarmHeader header

Position position 
Movement movement 
